backdoor-ctf-2015/crypto/rand/rand.py

```diff
@@ -0,0 +1,22 @@
+import urllib2
+import matplotlib.pyplot as plt
+from collections import defaultdict
+from cookielib import CookieJar
+
+cj = CookieJar()
+opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cj))
+
+poss = defaultdict(list)
+
+for tries in range(255):
+    url = opener.open("http://hack.bckdr.in/RAND/").read()
+    for i in range(0, len(url)):
+        poss[i].append(int(url[i], 16))
+
+plt.plot(poss.keys(), poss.values(), "ro")
+plt.show()
+
+flag = ""
+for i in range(999,1064):
```

Comment:
Should be 999,1063. Nice writeup!

I wasn't expecting this to be solved without the X-Paginate header being considered, but I missed the len() vector.